test/e2e/main_test.go

```diff
@@ -39,24 +39,29 @@ func TestMain(m *testing.M) {
 		exitCode int
 	)
 
+	if err = e2eframework.SplitFlags(); err != nil {
+		log.Error("Error parsing regular test flags:", err)
```

Comment:
You miss blank space here:
```suggestion
		log.Errorf("Error parsing regular test flags: %v", err)
```